sdk/resourcemanager/azure-resourcemanager-containerinstance/src/test/java/com/azure/resourcemanager/containerinstance/ContainerGroupTest.java

```diff
@@ -225,4 +228,135 @@ public void testBeginCreateWithFileShareVolume() {
         ContainerGroup containerGroup = acceptedContainerGroup.getSyncPoller().getFinalResult();
         Assertions.assertEquals(1, containerGroup.volumes().size());
     }
+
+    @Test
+    public void testCreateWithDnsNameLabelNoReuse() {
```

Comment:
Same with https://github.com/Azure/azure-sdk-for-java/pull/42795/files#r1830426052

Reply:
Fixed in the [commit#6e748a3](https://github.com/Azure/azure-sdk-for-java/pull/42795/commits/6e748a33351eeda68205b4a03d69db268dd3d0ed)